src/core/tsi/ssl_transport_security.c

```diff
@@ -49,6 +49,8 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 
+#include "src/core/support/env.h"
```

Comment:
Is there any reason not to use the pem_root_certs here as the flag? As
currently documented:

"""
   - pem_roots_cert is the NULL-terminated string containing the PEM
encoding of
     the client root certificates. This parameter may be NULL if the server
does
     not want the client to be authenticated with SSL.
"""

Thus, if the pem_roots_cert is non-NULL, the user wishes clients to be
authenticated with SSL, and we should force verification.

On Wed, Jul 1, 2015 at 11:29 AM, Geremy Condra <gcondra@google.com> wrote:

> Yep
>
> On Wed, Jul 1, 2015 at 1:58 AM, jboeuf <notifications@github.com> wrote:
>
>> In src/core/tsi/ssl_transport_security.c
>> <https://github.com/grpc/grpc/pull/2255#discussion_r33660069>:
>>
>> > @@ -49,6 +49,8 @@
>> >  #include <openssl/x509.h>
>> >  #include <openssl/x509v3.h>
>> >
>> > +#include "src/core/support/env.h"
>>
>> Re:ciphersuites. Do you mean the use of an environment variable here?
>>
>> https://github.com/grpc/grpc/blob/master/src/core/security/security_connector.c#L74
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/grpc/grpc/pull/2255/files#r33660069>.
>>
>
>